app.js

```diff
@@ -0,0 +1,20 @@
+let express = require('express');
+let app = express();
+let bodyParser = require('body-parser');
+
+import { parsedCookiesMW, parsedQueryMW } from './middlewares';
```

Comment:
Consider removing "MW" from the naming.
Think about your files as modules in Node.js ecosystem.
If you take a look at node_modules typical file structure and naming, it is generally a good practice to avoid such redundant naming.